plugins/registry.js

```diff
@@ -271,6 +273,10 @@ export default class PluginRegistry {
                 },
             });
             return function registerSubMenuItem(t, a, f) {
+                if (parentMenuId) {
+                    throw new Error('Submenus are currently limited to a single level.'); // eslint-disable-line no-console
```

Comment:
Do we need the `eslint-disable-line no-console` here?

Reply:
Oops, nope!
```suggestion
                    throw new Error('Submenus are currently limited to a single level.');
```